### PR TITLE
[bulk] validation: missing descriptions, HTML issues

### PR DESCRIPTION
**Pull request by LSchaufMS**
[bulk] validation: missing descriptions, HTML issues, table syntax.
This PR changes 23 files.

### CODE REVIEW

**Comment by LSchaufMS** [comment]
**Global effort to fix build validation errors**

@lindalu-MSFT 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: Skype/UCWA/ApplicationDashboard.md [hunk 1]
```diff
@@ -13,15 +17,16 @@ An [application](application_ref.md) resource represents a single application on
 
 The following table contains a representation of the **application** resource.
 
-
 **Property bag**
+
```
Comment (Roudabeh1348) [inline Skype/UCWA/ApplicationDashboard.md:21]: did you add an empty line purposefully or was it created due to changing the format?

Comment (LSchaufMS) [reply Skype/UCWA/ApplicationDashboard.md:21]: Add an empty line to cleanly separate this content from the code-block.

File: Skype/UCWA/conversationLogTranscript_ref.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+---
+title: conversationLogTranscript
+description: conversationLogTranscript Represents a transcript within a conversationLog.
```
Comment (Roudabeh1348) [inline Skype/UCWA/conversationLogTranscript_ref.md:3]: lowercase R in Represents

Comment (LSchaufMS) [reply Skype/UCWA/conversationLogTranscript_ref.md]: Updated.

File: Skype/UCWA/audioVideoNegotiation_ref.md [hunk 3]
```diff
@@ -1,14 +1,13 @@
 ---
 title: audioVideoNegotiation
-description: A resource that provides an outgoing audioVideo call its media answers. This resource appears only in the event channel or in a response to a request. It will not appear in a request.
+description: A resource that provides an outgoing audioVideo call its media answers. This resource appears only in the event channel or in a response to a request.
```
Comment (Roudabeh1348) [inline Skype/UCWA/audioVideoNegotiation_ref.md:3]: removed the last sentence intentionally? It will not appear in a request.

Comment (LSchaufMS) [reply Skype/UCWA/audioVideoNegotiation_ref.md:3]: Yes. The description was running a little long.

**Pull request merged**